### PR TITLE
Integrated autoplay button in video player

### PR DESCRIPTION
**Pull request by ubello64**
## Pull Request Type

- [ ] Bugfix
- [x] Feature Implementation
- [ ] Documentation
- [ ] Other

## Related issue
https://github.com/FreeTubeApp/FreeTube/issues/1181 

## Description
Adds a button that toggles Auto Play when in player mode, shows shortcut on hover

## Screenshots 
screengrab_ft

## Testing 
Tested functionality in full window and regular to ensure next video played

## Desktop
Ubuntu 22.04.3 LTS
FreeTube version: 0.19.1

## Additional context
This line in videoJS.css caused FreeTube to crash today, unsure of why this is
thiscausedft_tocrash

### CODE REVIEW

File: src/renderer/videoJS.css [hunk 1]
```diff
@@ -1383,11 +1387,11 @@ body.vjs-full-window {
   position: relative;
   top: -1px;
 }
-
-.video-js .vjs-play-control .vjs-icon-placeholder {
+/*
+.video-js .vjs-play-control .vjs-icon-placeholder {spinner
```
Comment (ChunkyProgrammer) [inline src/renderer/videoJS.css:1391]: Is it crashing because of the added "spinner"?

Comment (ubello64) [reply src/renderer/videoJS.css:1391]: sorry just caught that extra reference to 'spinner' that shouldn't be there - FreeTube is now fine

**Comment by PikachuEXE** [comment]
I suggest you to use a different branch name next time
`gh pr checkout 4335` will try to switch to `development` but can't checkout easily :S

**Comment by PikachuEXE** [comment]
Strange focus style (happens after click, not related to autoplay enable state)
image

I could not tell from that button whether autoplay is enabled or not
image

File: src/renderer/videoJS.css [hunk 2]
```diff
@@ -111,10 +111,14 @@
   font-family: VideoJS;
   font-weight: normal;
   font-style: normal;
+  margin-top: 9px;
 }
 .vjs-icon-spinner:before {
   content: "\f10b";
 }
+.vjs-button > .vjs-icon-spinner:before {
+ font-size: 1.66666666667em;
```
Comment (PikachuEXE) [inline src/renderer/videoJS.css:120]: Minor indent issue but you should use an editor/update the editor you are using to read and enforce rules in https://github.com/FreeTubeApp/FreeTube/blob/development/.editorconfig file
See https://editorconfig.org
```suggestion
  font-size: 1.66666666667em;
```

**Comment by ubello64** [comment]
Closing pull request - don't want to cause potential merge conflict. Feature also does not help end user determine whether or not if Auto play is on. Will open another pull request when the main issue is resolved

**Pull request closed without merging**